docs-src/references/commands.md

```diff
@@ -50,27 +50,55 @@ This Command is triggered by a call to spawn a [Child Workflow Execution](/conce
 - Awaitable: Yes, a Workflow Execution can await on the action resulting from this Command.
 - Corresponding Event: [ChildWorkflowExecutionStarted](/references/events/#childworkflowexecutionstarted)
 
+:::note
+
+OSS users cannot have more than 50000 pending Child Workflows.
```

Comment:
Is this saying the following:

Within all my Workflow code that is executing, if I have loops (for example) that spin off x number of Child Workflows - at any given time I should be wary not to call that API too much as I don't want to possibly exceed 5000 pending ChildWorkflows across my Namespace (or is it per Account ?) - and PENDING in this instance means that the Commands have not yet been processed into corresponding Events?

If so, let us clarify that -